arc-theme-upgrade

```diff
@@ -0,0 +1,20 @@
+#/!bin/bash
```

Comment:
What you want to say with the script will accidentally corrupt the package database?